modules/core/include/opencv2/core.hpp

```diff
@@ -1118,6 +1118,13 @@ CV_EXPORTS_W void flip(InputArray src, OutputArray dst, int flipCode);
  */
 CV_EXPORTS_W void flipND(InputArray src, OutputArray dst, int axis);
 
+/** @brief Broadcast the given Mat to the given shape.
+ * @param src input array
+ * @param dst output array that has the given shape
+ * @param shape target shape
+ */
+CV_EXPORTS_W void broadcast_to(InputArray src, const std::vector<int>& shape, OutputArray dst);
```

Comment:
Please use OpenCV naming conventions. Should be `broadcastTo`

Reply:
@vpisarev Should it be `Mat` class method?

Reply:
> Please use OpenCV naming conventions. Should be broadcastTo

Done.

Reply:
> Please use OpenCV naming conventions. Should be broadcastTo

Done.